docs/api.rst

```diff
@@ -9233,6 +9233,46 @@ Adding EVM accounts to all EVM chains
    :statuscode 500: Internal rotki error
    :statuscode 502: Remote error occured when attempted to connect to an Avalanche or Polkadot node and only if it's the first account added. Check message for details.
 
+
+.. http:post:: /api/(version)/blockchains/evm/accounts
+
+   Doing a POST on the EVM accounts endpoint will re-detect evm accounts on all supported chains. Rotki will go through already added addresses and for each address, if it is an EOA (not a smart contract) and has activity on some chain, will add it to that chain. Newly detected accounts will be returned.
+
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      POST /api/1/blockchains/evm/accounts HTTP/1.1
+      Host: localhost:5042
+
+      {}
+
+      .. note::
+        This endpoint can also be queried asynchronously by using ``"async_query": true``
```

Comment:
You have written the note inside the http request 

![image](https://user-images.githubusercontent.com/5068010/218795584-fc2a7557-5e52-4487-a182-cde085b37386.png)

Reply:
lol. thanks